supplementary_style_guide/style_guidelines/support.adoc

```diff
@@ -1,6 +1,52 @@
 [[support]]
 = Support
 
+[[developer-preview-guidance]]
+== Developer Preview
+
+Developer Preview software provides early access to a technology, component, or feature, in advance of its possible inclusion in a Red Hat product offering. This enables customers to test the functionality and provide feedback during the development process. Documentation is not required for Developer Preview software. Documentation that is provided for Developer Preview software is subject to change or removal at any time, and testing is limited. Red Hat might provide ways to submit feedback on Developer Preview software without an associated SLA.
```

Comment:
```suggestion
Developer Preview software provides early access to a technology, component, or feature, in advance of its possible inclusion in a Red Hat product offering. Customers can use Developer Preview software to test functionality and provide feedback during the development process. Documentation is not required for Developer Preview software, but if documentation is provided, it is subject to change or removal at any time. Also, testing is limited for Developer Preview software. Red Hat might provide ways to submit feedback on Developer Preview software without an associated SLA.
```
- Same comment as above re `Red Hat` and non-breaking spaces.
- Same comment as above re anthropomorphism. Also "This" doesn't have a clear antecedent.
- I think this needs better logical flow. Consider my edits as one possible way to improve the flow.

Maybe consider just using the identical wording to the Glossary entry??

Reply:
I went with your suggested edit thanks.